code/modules/games/cards.dm

```diff
@@ -0,0 +1,469 @@
+	user.visible_message("<span class = 'notice'>[user] pick up the deck.</span>")
+
+/obj/item/weapon/deck/verb/draw_card(mob/user as mob)
+
+	set category = "Object"
+	set name = "Draw"
+	set desc = "Draw a card from a deck."
+	set src in view(1)
+
+	var/mob/living/carbon/human/M = user
+
+	if(user.incapacitated() || !Adjacent(user))
+		return
+
+	if(!cards.len)
+		to_chat(user,"<span class='notice'>There are no cards in the deck.</span>")
+		return
+
+	var/obj/item/weapon/hand/H = M.is_in_hands(/obj/item/weapon/hand)
+	if(H && !(H.parentdeck == src))
+		to_chat(user,"<span class='warning'>You can't mix cards from different decks!</span>")
+		return
+
+	if(!H)
+		H = new(get_turf(src))
+		user.put_in_hands(H)
+
+	var/datum/playingcard/P = cards[1]
+	H.cards += P
+	cards -= P
+	H.parentdeck = src
+	H.update_icon()
+	user.visible_message("<span class='notice'>[user] draws a card.</span>")
+	to_chat(user,"<span class='notice'>It's the [P].</span>")
+
+/obj/item/weapon/deck/verb/deal_card(mob/user as mob)
+
+	set category = "Object"
+	set name = "Deal"
+	set desc = "Deal a card from a deck."
+	set src in view(1)
+
+	if(user.incapacitated() || !Adjacent(user))
+		return
+
+	if(!cards.len)
+		to_chat(user,"<span class='notice'>There are no cards in the deck.</span>")
+		return
+
+	var/list/players = list()
+	for(var/mob/living/player in viewers(3))
+		if(!player.incapacitated())
+			players += player
+	//players -= usr
+
+	var/mob/living/M = input("Who do you wish to deal a card to?") as null|anything in players
+	if(!user || !src || !M) return
+
+	deal_at(user, M, 1)
+
+/obj/item/weapon/deck/verb/deal_card_multi()
+
+	set category = "Object"
+	set name = "Deal Multiple Cards"
+	set desc = "Deal multiple cards from a deck."
+	set src in view(1)
+
+	if(usr.incapacitated() || !Adjacent(usr))
+		return
+
+	if(!cards.len)
+		to_chat(usr,"<span class='notice'>There are no cards in the deck.</span>")
+		return
+
+	var/list/players = list()
+	for(var/mob/living/player in viewers(3))
+		if(!player.incapacitated())
+			players += player
+	var/maxcards = max(min(cards.len,10),1)
+	var/dcard = input("How many card(s) do you wish to deal? You may deal up to [maxcards] cards.") as num
+	if(dcard > maxcards)
+		return
+	var/mob/living/M = input("Who do you wish to deal [dcard] card(s)?") as null|anything in players
+	if(!usr || !src || !M || !Adjacent(usr)) return
+
+	deal_at(usr, M, dcard)
+
+/obj/item/weapon/deck/proc/deal_at(mob/user, mob/target, dcard) // Take in the no. of card to be dealt
+	var/obj/item/weapon/hand/H = new(get_step(user, user.dir))
+	var/i
+	for(i = 0, i < dcard, i++)
+		H.cards += cards[1]
+		cards -= cards[1]
+		H.parentdeck = src
+		H.concealed = TRUE
+		H.update_icon()
+	if(user==target)
+		user.visible_message("<span class = 'notice'>[user] deals [dcard] card(s) to \himself.</span>")
+	else
+		user.visible_message("<span class = 'notice'>[user] deals [dcard] card(s) to [target].</span>")
+	H.throw_at(get_step(target,target.dir),10,1,H)
```

Comment:
inb4 a traitor deck of cards with a dangerous throwforce and decent range, a-la gambit

Reply:
@Crazylemon64 (thats already a thing, and its glorious)